mrbgems/mruby-objectspace/test/objectspace.rb

```diff
@@ -1,4 +1,3 @@
-# coding: cp932
```

Comment:
I am sorry I think I introduced this initially. I did not catch that `ruby-mode` adds the encoding comment.